astropy/cosmology/_io/tests/test_model.py

```diff
@@ -144,7 +144,7 @@ def test_fromformat_model_subclass_partial_info(self):
         Test writing from an instance and reading from that class.
         This works with missing information.
         """
-        pass  # there's no partial information with a Model
+        # there's no partial information with a Model
```

Comment:
Please move this and the other comments to the docstring. No point having a comment without any code. 🙃

Reply:
Thanks for the suggestion! I have added the comments to the docstrings.